src/components/Layout/Header.jsx

```diff
@@ -29,46 +29,36 @@ import classes from "./HeaderTabs.module.css";
 import { Link, useLocation, useNavigate } from "react-router-dom";
 import logo from "../../assets/logo-new.png";
 import useToast from "../../hooks/useToast";
-import { getUser } from "../../service/users";
 import useFetch from "../../hooks/useFetch";
-import { removeToken } from "../../util/security";
 
 export const Header = ({ user, setUser }) => {
```

Comment:
user object passed by props from `Header.jsx` so simply can destruct it and pass to API body